lib/model/compose.dart

```diff
@@ -5,6 +5,31 @@ import 'internal_link.dart';
 import 'narrow.dart';
 import 'store.dart';
 
+/// The available user wildcard mention options,
+/// known to the server as [canonicalString].
+///
+/// There isn't an API doc for these yet, but see the values in the Help Center:
+///   https://zulip.com/help/mention-a-user-or-group
+///   https://zulip.com/help/dm-mention-alert-notifications
+enum WildcardMentionOption {
+  all(canonicalString: 'all'),
+  everyone(canonicalString: 'everyone'),
+  channel(canonicalString: 'channel'),
+  // Deprecated in FL 247. Empirically, current servers (FL 330)
+  // still parse "@**stream**" in messages though.
+  stream(canonicalString: 'stream'),
+  topic(canonicalString: 'topic'); // New in FL 224.
```

Comment:
```suggestion
  topic(canonicalString: 'topic'); // TODO(server-8): New in FL 224.
```

The TODO-server comment is how we make sure we can systematically clean out these historical comments in the future when they're no longer relevant. Basically all references to a Zulip feature level (as well as anything else that will become obsolete when we drop support for some old Zulip Server version) should have a TODO-server comment that will make sure we find it.